### ISSUE TITLE
Column usage description in custom widget

### ISSUE THREAD

**Issue by yohanboniface** [gristlabs/grist-core#245]
Hi!

While playing with widgets, I feel the need to be more descriptive on the columns we ask to the user.

Eg. here (a customized version of the map widget), I'd like to add a small description about why the two optional columns are asked and what they will be used for:

![image](https://user-images.githubusercontent.com/146023/185190104-2bc369a4-1e7e-4bd8-9fca-383911e47ee6.png)

May look something like this:

![image](https://user-images.githubusercontent.com/146023/185191193-fd6afc0d-b77b-4aae-ad17-d9177e773e28.png)

Would that make sense or am I misusing something here ?

Thank for your inputs! :)

**Comment by paulfitz** [comment]
Makes sense to me @yohanboniface. Perhaps the column mapping step from https://support.getgrist.com/widget-custom/#column-mapping could be tweaked so that, along with `name`/`title`/`optional`/`type`, an optional `description` could be given. Plain text should be simple; any markup might need some thinking about since it would be inserted outside the widget iframe and could cause new kinds of mischief in situations of less trust between widget writer and document creator. What do you think @berhalak?

**Comment by berhalak** [comment]
Yes, this is a good idea. Actually, we are working on a general solution for showing some help text, description or tooltips with some explenation for various UI elements, so this can play nicely with it.
